ncard_app/static/style.css

```diff
@@ -145,7 +145,7 @@ li.page-item > a:hover {
     box-shadow: 0 0 0 0.2rem rgba(40, 167, 69, 0.25);
 } 
 
-th#award-table {
+th#col-width-350 {
```

Comment:
Use a class name rather than an ID for this style, since you're applying it to multiple elements